backend/requirements.txt

```diff
@@ -3,7 +3,7 @@ bitmath==1.3.1.2
 celery==4.4.0
 clkhash==0.15.1
 colorama==0.4.1 # required for structlog
-connexion[swagger-ui]==2.6
+connexion==2.6.0
```

Comment:
I thought we saw warnings logged if we didn't include `swagger-ui`?

Reply:
two options:
1. disable swagger-ui. 
```
con_app.add_api(pathlib.Path("openapi.yaml"),
                base_path='/',
                options={'swagger_ui': False},
                strict_validation=config.CONNEXION_STRICT_VALIDATION,
                validate_responses=config.CONNEXION_RESPONSE_VALIDATION)
```
2. use swagger_ui. For this, we have to solve two problems. First the default url to the spec is not working in our case.
```
con_app.add_api(pathlib.Path("openapi.yaml"),
                base_path='/',
                options={'swagger_ui_config': {'url': '/api/v1/openapi.json'}},
                strict_validation=config.CONNEXION_STRICT_VALIDATION,
                validate_responses=config.CONNEXION_RESPONSE_VALIDATION)
```
Yay. Secondly, connexion removes the server description/url. (See above) Therefore you cannot actually use the UI to hit the endpoints.

Because 2 doesn't work, I favor 1.

Reply:
Also in favor of option 1 👍